build.gradle.kts

```diff
@@ -0,0 +1,45 @@
+plugins {
+    `java-library`
+    kotlin("jvm") version "1.9.0"
+    kotlin("plugin.allopen") version "1.9.10"
+    kotlin("plugin.spring") version "1.9.10"
+    kotlin("plugin.noarg") version "1.9.10"
+    kotlin("plugin.jpa") version "1.9.10"
+    id("io.gitlab.arturbosch.detekt") version "1.23.1"
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    api("org.springframework.boot:spring-boot-starter-data-jpa:3.1.2")
+    api("org.springframework.boot:spring-boot-starter-security:3.1.2")
+    api("org.springframework.boot:spring-boot-starter-web:3.1.2")
+    api("io.jsonwebtoken:jjwt:0.9.1")
+    api("org.springframework.boot:spring-boot-starter-validation:3.1.2")
+    api("org.jetbrains.kotlin:kotlin-stdlib:1.9.0")
```

Comment:
unnecessary dependency. You have already applied kotlin plugin `kotlin("jvm") version "1.9.0"`, so the dependency already added implicitly

Reply:
Done!